packages/vite-plugin-kit-routes/src/lib/plugin.ts

```diff
@@ -942,28 +942,26 @@ ${objTypes
       }
 
       // do the stuff
-      const child = spawn(options.post_update_run, { shell: true })
+      const child = spawnSync(options.post_update_run, { shell: true })
 
       // report things
       if (shouldLog('post_update_run', options)) {
-        child.stdout.on('data', data => {
-          if (data.toString()) {
-            log.info(data.toString())
-          }
-        })
+        const stdout = child.stdout.toString()
```

Comment:
With this, I think that you see the logs only at 'the end". Right? 
I mean the all process happen, and then it print the result, it's not streamed.

If it's really short, I dont' mind, but if it's a bit longer, I'm not a huge fan...

But why not! Let me know if you know :)

Reply:
Yeah, that's probably right. I'll experiment with waiting for the exit of the sub process.

Reply:
Indeed, it works by awaiting a promise.